scripts/training_loop.py

```diff
@@ -242,7 +197,15 @@ def train(self):
                 message = f"Taking step {episode_timesteps} of Episode {episode_num} with Total T {total_step_counter} \n"
                 logging.info(message)
 
-                action = self.agent.select_action_from_policy(state, noise_scale=self.noise_scale)  # algorithm range [-1, 1]
+                if (self.algorithm == ALGORITHMS.TD3.value):
+                    action = self.agent.select_action_from_policy(state, noise_scale=self.noise_scale)  # returns a 1D array with range [-1, 1], only TD3 has noise scale
```

Comment:
This should be handled elsewhere/within the networks themselves - @retinfai / @dvalenciar